src/app/recipes/store/recipe.effect.ts

```diff
@@ -42,6 +51,50 @@ export class RecipeEffects {
     })
   );
 
+  @Effect()
+  AddRecipeToDataBase = this.actions$.pipe(
+    ofType(RecipesActions.ADD_RECIPE_TO_DB),
+    switchMap((actionData: RecipesActions.AddRecipeToDataBase) => {
+      return this.http.post<Recipe>(
+        environment.DataBaseUrl + 'recipes',
+        actionData.payload
+      );
+    }),
+    map((recipe) => {
+      console.log(recipe); //добавить handle errors
```

Comment:
Errors handling adds in `catchError` operator scope

Reply:
added